pilot/pkg/serviceregistry/aggregate/controller.go

```diff
@@ -360,10 +372,25 @@ func (c *Controller) AppendWorkloadHandler(f func(*model.WorkloadInstance, model
 
 // GetIstioServiceAccounts implements model.ServiceAccounts operation
 func (c *Controller) GetIstioServiceAccounts(svc *model.Service, ports []int) []string {
+	out := map[string]struct{}{}
```

Comment:
Can you add a small comment to the doc with an example of how the result will look like ? Is it a list of spifee URLs ? Can it include other types of identifiers ( like email or URL SANs from annotations)  ?

Reply:
added the comment.

what's the reason though? just to have an entry place for dev to understand the overall returned string formats? this method itself does very small things, doc is more like a summarize of logic in different parts.

Reply:
Yes, I was confused - I assume others too.